Examples/test-suite/common.mk

```diff
@@ -121,6 +121,7 @@ CPP_TEST_CASES += \
 	apply_strings \
 	argcargvtest \
 	argout \
+	csharp_argument_defaults_feature \
```

Comment:
Delete this line as the testcase has been added to Examples/test-suite/csharp/Makefile.in.